lms/static/sass/base/_reset.scss

```diff
@@ -96,4 +96,5 @@ td { vertical-align: top; }
   @page { margin: 0.5cm; }
   p, h2, h3 { orphans: 3; widows: 3; }
   h2, h3 { page-break-after: avoid; }
+  .donotprint { display:none !important; }
```

Comment:
A few notes here @jolyonb  - It looks like you're not using .donotprint here, and I agree with sarina that this seems an unnecessary workaround to include behind the scenes. 

nit / suggestion: That being said, in a similar vein, I would suggest adding an extend to _extends.scss (within lms/static/sass/base) called %ui-print-excluded (or something similar). That way, if we have any other print specific styles in the future, we can quickly separate the intent of the display: none rule.